test/statistics.jl

```diff
@@ -96,6 +100,9 @@ end
 @test_approx_eq var([1], 1; mean=[2], corrected=false) [1.0]
 
 @test var(1:8) == 6.
+@test varm(1:8,1) == var(1:8)
```

Comment:
It's coming from here https://github.com/JuliaLang/julia/blob/master/base/statistics.jl#L180 which might be totally crazy?